crypto/nrf_cc310_platform/include/nrf_cc3xx_platform_kmu.h

```diff
@@ -144,5 +219,53 @@ int nrf_cc3xx_platform_kdr_load_key(uint8_t key[16]);
 
 #endif // defined(NRF52840_XXAA)
 
+/** @brief Function to use CMAC to derive a key stored in KMU/KDR
+ *
+ * @details The KDF is using a PRF function described in the Special publication
+ *          800-108: Recommendation for Key Derivation Using Pseudorandom Functions
+ *          https://csrc.nist.gov/publications/detail/sp/800-108/final.
+ *
+ *          This algorithm is described in chapter 5.1 - KDF in Counter Mode
+ *
+ *          The format of the PRF (the input) is as follows:
+ *          PRF (KI, i || Label || 0x00 || Context || L)
+ *
+ *          KI: The Key derivation key
+ *          i : The counter value for each iteration of the PRF represented
+ *              as one byte.
+ *          label: A string identifying the purpose of the derived key
+ *                 that is up to 64 bytes long.
+ *          0x00: a single byte delimiter.
+ *          Context: Fixed information about the derived keying material
+ *                   that is up to 64 bytes long.
+ *          L : The length of derived key material in bits represented as two
+ *              bytes.
+ *
+ * @note On nRF52840 only slot_id == 0 is valid, pointing to the
+ *       Kdr key (also known as a HUK key) loaded into the CryptoCell.
+ *
+ * @param   slot_id         Identifier of the key slot.
+ * @param   keybits         Key size in bits.
+ * @param   label           Label to use for KDF.
+ * @param   label_size      Size of the label to use for KDF.
+ * @param   context         Context info to use for KDF.
+ * @param   context_size    Context info size to use for KDF.
+ * @param   output          Output buffer.
+ * @param   output_size     Size of output buffer in bytes.
```

Comment:
```suggestion
 * @param   output_size     Size of the output buffer in bytes.
```

Reply:
Will be fixed, also will be added to 0.9.10 release as well

Reply:
Done